### ISSUE TITLE
Mutliple plots in single HTML

### ISSUE THREAD

**Issue by nipunbatra** [wrobstory/bearcart#3]
It would be good if multiple plots could come in a single html file. The advantage would be that we get sort of inline plots like in IPython Notebook, but the added advantage is that they are interactive. Would certainly be valuable giving demos during research talks!

**Comment by wrobstory** [comment]
Yep, that would be valuable. This is kind of tied into iPython notebook integration- stay tuned, I'm definitely going to be working on this over the next month.

**Comment by raddy** [comment]
I'd also be very interested in this feature; I generate a LOT of TS charts in pandas -- this would potentially be awesome.

**Comment by wrobstory** [comment]
Ok- it might take a little bit of work to build the looping into the jinja template so that you can stack charts, and possibly some queue.js-ing because of the asynchronous data load. I'll leave this open until I get to it, but if either of you want to submit a PR (including a test or two), I'll be happy to merge it into the codebase.
